bin/main.ml

```diff
@@ -901,6 +902,56 @@ let subst =
          )
   , Term.info "subst" ~doc ~man)
 
+let utop =
+  let doc = "Load library in utop" in
+  let man = [ (* TODO *) ] in
+  let go common dir ctx_name args =
+    let utop_target = dir |> Path.of_string |> Utop.utop_exe |> Path.to_string in
+    set_common common ~targets:[utop_target];
+    let log = Log.create () in
+    let (build_system, context, utop_path) =
+      (Main.setup ~log common >>= fun setup ->
+       let context =
```

Comment:
Sorry, looking at this again we already have something similar for `jbuilder exec`. Can you factorize the code between these two command and use the same behavior?